src/pretix/base/services/cart.py

```diff
@@ -512,7 +512,10 @@ def extend_expired_positions(self):
             if cp.is_bundled:
                 bundle = cp.addon_to.item.bundles.filter(bundled_item=cp.item, bundled_variation=cp.variation).first()
                 if bundle:
-                    listed_price = bundle.designated_price or Decimal('0.00')
+                    if cp.addon_to.voucher_id and cp.addon_to.voucher.all_bundles_included:
+                        listed_price = Decimal('0.00')
+                    else:
+                        listed_price = bundle.designated_price or Decimal('0.00')
```

Comment:
Is that `or`-part still needed? `designated_price` has a default of `Decimal('0.00')` so I guess this is not needed – if it is still needed, `bundle.designated_price` is used in other places where it should have an `or Decimal('0.00')`.

```suggestion
                        listed_price = bundle.designated_price
```

Reply:
I was unable to trace that back, the or-part existed since we added bundles. I don't see why we need it either.